Staging -> production: subscriptions batch update script and GTC-3094

solomon-negusse opened the pull request:
Subscriptions updated in staging and here are the metrics from production dry run. The failures are all from missing subscriptions.
🔎  Validation Metrics:
🌍  Total GADM areas with Subscriptions: 8033
✅  Total GADM subscriptions updated Successfully: 7356
❌  Total failed GADM subscription updates: 677
📈  GADM subscriptions success rate: 91.57%
✅  Total subscriptions updated Successfully: 40384
❌  Total failed subscription updates: 1776
📈  Total subscriptions success rate: 95.79%
💡 [Dry Run] No changes were saved to the database.

solomon-negusse committed:
add script to batch update gadm subscriptions

solomon-negusse committed:
add area id to all subscriptions

solomon-negusse committed:
update both gadm and user area subscriptions

gtempus reviewed (approved):
Looks good, @solomon-negusse! 👍

gtempus committed:
:broom: chore: turn off koa and bunyan logging during test runs

gtempus committed:
feat(Area): look for gadm version in payload when saving or updating

- Default to gadm 3.6 if there is no adminVersions history
- Saving/Updating a specific admin version will return that same version in the representation

gtempus commented:
# GTC-3094 Testing Steps
Create an new administrative boundary Area with a 4.1 source:
`POST /v2/area`
```JSON
{
  "name": "Honduras",
  "admin": {
    "adm0": "HND",
    "source": {
      "provider": "gadm",
      "version": "4.1"
    }
  }
}
```

Verify that the returned representation includes the GADM 4.1 source.

---

Update the newly created administrative boundary Area with a 3.6 source:
`PATCH /v2/area/:id`
```JSON
{
  "name": "Honduras",
  "admin": {
    "adm0": "HND",
    "source": {
      "provider": "gadm",
      "version": "3.6"
    }
  }
}
```

Verify that the returned representation includes the GADM 3.6 source.

----

Get the list of your Areas and ensure the newly created administrative boundary is listed and that it is showing the GADM 3.6 information (currently the default for backward compatibility)

`GET /v2/area`

Partial Response:
```JSON
{
  "name": "Honduras",
  "admin": {
    "adm0": "HND",
    "source": {
      "provider": "gadm",
      "version": "3.6"
    }
  }
}
```
